### PR TITLE
Multilanguage version with gettext

### PR DESCRIPTION
**Pull request by mayersre**
This is the multilanguage version with gettext. German and english translations are provided for all of the GUI. Language can be selected in the new settings menu and works after restart of app. Build file for cx_freeze on window was extended to copy all new files into a build. Build was tested on Windows 7 and 8.1.

### CODE REVIEW

**Comment by jowr** [comment]
Looks good! Are the `*.pot~` files needed or can they be removed from the commit? It looks like they are some kind of backup...

File: cpgui_basics.py [hunk 1]
```diff
@@ -265,7 +275,7 @@ def Update(self):
                 molmassstring='  N/A '
             if molmassstring=='' :
                 molmassstring='%6.2f'%(molmass*1000)
-            self.Ausgabetext+='Molar mass           :%s  \n'%molmassstring
+            self.Ausgabetext+=_('Molar mass           :%s  \n')%molmassstring
```
Comment (ibell) [inline cpgui_basics.py]: Please keep the code in the codebase in english

**Comment by mayersre** [comment]
Am 24.03.2015 um 17:44 schrieb Jorrit Wronski:

> Looks good! Are the |*.pot~| files needed or can they be removed from 
> the commit? It looks like they are some kind of backup...
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/CoolProp/GUI/pull/3#issuecomment-85593394.
> 
> The files with asciitilde at the end can be deleted.
> I did not see them, but I edited some files with vim and it left the 
> backups.

the .pot files are generated with pygettext.py
the .mo files are generated with poedit

/Reiner

File: cpgui_basics.py (hunk 1 above)
Comment (mayersre) [inline cpgui_basics.py]: I will change that on the next commit/push

Ausgabetext => OutputText

/Reiner

Am 24.03.2015 um 17:46 schrieb Ian Bell:

Comment (ibell) [inline cpgui_basics.py]: Can you remove the .pot~ files from the PR please?

**Comment by mayersre** [comment]
I removed the *.pot~ files from my project and changed the german variable name to statetext

**Comment by ibell** [comment]
:) Thanks

**Pull request merged**